lib/rucio/client/downloadclient.py

```diff
@@ -1416,7 +1428,7 @@ def _check_output(self, output_items):
 
     def _send_trace(self, trace):
         """
-        Checks if sending trace is allowed and send the trace.
+        If reference to external list of traces given, traces are appended there.
```

Comment:
Deleted the wrong function description? (:

Reply:
That was just a test whether you notice :-D